packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -673,6 +673,26 @@ export class PrismicRepositoryManager extends BaseManager {
 		}
 	}
 
+	async setDefaultMasterLocale(): Promise<void> {
+		const repositoryName = await this.project.getRepositoryName();
+
+		const url = new URL("repository/locales", API_ENDPOINTS.LocaleService);
+		url.searchParams.set("repository", repositoryName);
+
+		const res = await this._fetch({
+			url,
+			method: "POST",
+			body: {
+				id: "en-us",
+				isMaster: true,
+			},
+		});
+
+		if (!res.ok) {
```

Comment:
⚠️ You should take care of all potential status to make it clear when debugging.

```
                         case 400:
			case 401:
				throw new UnauthenticatedError();
			case 403:
				throw new UnauthorizedError();
```

Reply:
done